doc/index.rst

```diff
@@ -50,36 +50,36 @@ If you have any questions, feedback, or feature requests, please `let us know on
       **Tutorials**
       ^^^^^^^^^^^^^
 
-      Tutorials are the place where new users can learn the basics of how to use
-      the library. No prior knowledge of differential privacy is required!
+      For new users to learn the basics of how to use the library.
+      No prior knowledge of differential privacy is required!
 
    .. grid-item-card::
-      :img-top: /images/index_howto_guides.svg
+      :img-top: /images/index_topic_guides.svg
       :class-img-top: intro-card-icon
-      :link: howto-guides/index
+      :link: topic-guides/index
       :link-type: doc
-      :link-alt: How-to guides
+      :link-alt: Topic guides
       :text-align: center
 
-      **How-to guides**
+      **Topic guides**
       ^^^^^^^^^^^^^^^^
 
-      How-to guides are step-by-step instructions on how to install and
-      troubleshoot the library locally or on a cloud platform.
+      To dive deeper into specific aspects of the library, and understand in
+      more detail how it works behind the scenes.
 
    .. grid-item-card::
-      :img-top: /images/index_topic_guides.svg
+      :img-top: /images/index_howto_guides.svg
       :class-img-top: intro-card-icon
-      :link: topic-guides/index
+      :link: deployment/index
       :link-type: doc
-      :link-alt: Topic guides
+      :link-alt: How-to guides
       :text-align: center
 
-      **Topic guides**
+      **Deployment**
       ^^^^^^^^^^^^^^^^
 
-      Topic guides dive deeper into specific aspects of the library, and explain in
-      more detail how it works behind the scenes.
+      Step-by-step instructions on how to deploy and troubleshoot Tumult
+      Analytics in a variety of environments.
```

Comment:
```suggestion
      Find step-by-step instructions on how to deploy and troubleshoot Tumult
      Analytics in a variety of environments.
```

I feel less strongly about these last two changes, but having parallel construction (each description starts with a verb and explains what you're going to do if you click this) seems nice.